CMakeLists.txt

```diff
@@ -12,7 +12,7 @@ project(redpanda LANGUAGES CXX)
 list(APPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/cmake)
 
 set(CMAKE_C_STANDARD 11)
-set(CMAKE_CXX_STANDARD 20)
```

Comment:
I still need to do this in vtools, this is for the public build which isn't maintained.